packages/client/src/components/ToolItem/ToolItem.component.js

```diff
@@ -86,10 +87,7 @@ export const ToolItem = ({
           <h6 className="product-title">{title}</h6>
         </div>
         <div className="title-image">
-          <img
-            src="/assets/images/tool-pic-stinky-fish.png"
-            alt="stinky-fish"
-          />
+          <img src={picture} alt="stinky-fish" />
```

Comment:
The `alt="stinky-fish"` will not fit every tool, so I would suggest working with the picture URL to extract the name somewhat dynamically.
Something like:
Take `picture` string, cut away the part of `/assets/images/tool-pic-` and the `.png` at the end, and do something like "Picture of tool" + `result`, where result is what you have after processing the picture string. Hope that makes sense 😅

Reply:
Yes, I agree with you. It makes sense :) Fixed already .